crates/burn-import/src/onnx/op_configuration.rs

```diff
@@ -799,10 +799,13 @@ pub fn pad_config(node: &Node) -> PadConfig {
     }
     fn get_constant_value(node: &Node) -> f32 {
         // TODO: support int, boolean
-        match &node.inputs[2].value {
-            Some(Data::Float32s(shape)) => shape.first().unwrap().to_owned(),
-            _ => 0.0,
-        }
+        node.inputs
+            .get(2)
+            .and_then(|input| match &input.value {
+                Some(Data::Float32s(shape)) => shape.first().cloned(),
+                _ => None,
```

Comment:
We need to improve the logic a bit. The implementation seems to have a bug. 

If get(2) is None, then default to 0.0.
If get(2) is Some with Float32s then return the first item from `constant_value` (btw `shape` should be renamed to `constant_value`. (I also recommend handling Float16s, Float64s with casting, just in case)
If get(2) is Some with other that floats, then panic (int or boolean probably). Currently I guess it's not fully supported.